test/state/precompiles.hpp

```diff
@@ -1,13 +1,34 @@
 // evmone: Fast Ethereum Virtual Machine implementation
 // Copyright 2022 The evmone Authors.
 // SPDX-License-Identifier: Apache-2.0
-
 #pragma once
 
 #include <evmc/evmc.hpp>
 #include <optional>
 
 namespace evmone::state
 {
+/// The total number of known precompiles ids, including 0.
+inline constexpr std::size_t NumPrecompiles = 10;
+
+enum class PrecompileId : uint8_t
+{
+    ecrecover = 0x01,
+    sha256 = 0x02,
+    ripemd160 = 0x03,
+    identity = 0x04,
+    expmod = 0x05,
```

Comment:
Why expmod and not modexp? Yellowpaper? I know clients disagree on the naming.

Reply:
IDK, maybe because addmod and mulmod.